Modules/Session/classes/class.ilObjSessionGUI.php

```diff
@@ -44,6 +44,7 @@ class ilObjSessionGUI extends ilObjectGUI implements ilDesktopItemHandling
     public ilTree $tree;
     protected ilAccessHandler $access;
     protected ilToolbarGUI $toolbar;
+    //TODO PHP8-REVIEW: Missing property's declaration?
     protected $ilErr;
```

Comment:
`$ilErr` should be renamed to `$error` with type `ilErrorHandling`, because this property has been renamed in `ilObjectGUI` in the meantime.